packages/ndla-ui/src/Resource/resourceComponents.tsx

```diff
@@ -90,35 +96,45 @@ const TagCounterWrapper = styled.p`
 
 interface TagListProps {
   tags?: string[];
+  tagLinkPrefix?: string;
 }
-
-export const TagList = ({ tags }: TagListProps) => {
+export const TagList = ({ tags, tagLinkPrefix }: TagListProps) => {
   if (!tags) return null;
   return (
     <StyledTagList>
       {tags.map((tag, i) => (
-        <StyledTagListElement key={`tag-${i}`}>{tag}</StyledTagListElement>
+        <StyledTagListElement key={`tag-${i}`}>
+          <StyledSafeLink
+            onClick={(e: MouseEvent<HTMLAnchorElement | HTMLElement>) => e.stopPropagation()}
+            to={`${tagLinkPrefix ? tagLinkPrefix : ''}/${tag}`}>
+            {tag}
+          </StyledSafeLink>
```

Comment:
Eg tenkte meir nokke sånt for å unngå lenke dersom det ikkje er ønskelig å lage lenke i det heile tatt:

```suggestion
          {tagLinkPrefix ? (<StyledSafeLink
            onClick={(e: MouseEvent<HTMLAnchorElement | HTMLElement>) => e.stopPropagation()}
            to={`${tagLinkPrefix ? tagLinkPrefix : ''}/${tag}`}>
            tag
          </StyledSafeLink>) : {tag}
```

Om du er uenig må du berre seie ifra.

Reply:
Det er et godt forslag, men eneste jeg funderer på er om vi noen sinne vil komme i en situasjon der man ikke skal kunne klikke på tagsene og linkes til en tag-side? Mister ikke da tagsene verdien?

Reply:
Jo, det er jo et godt poeng. Så lenge det er tags bør dei kunne klikkes på.

Reply:
![bitmoji](https://sdk.bitmoji.com/render/panel/4f211e34-f0a8-4ad4-b294-4aadf9aa76b7-5e0c4d43-55bc-4daf-ac97-c7042ea1a178-v1.png?transparent=1&palette=1&width=246)å